### ISSUE TITLE
inconsistency between bug flags

### ISSUE THREAD

**Issue by raphaeldussin** [NOAA-GFDL/MOM6#237]
some bugfix runtime parameters use `FIX_SOME_BUG = True` while the majority use the form `SOME_OTHER_BUG = False`,
which can lead to confusion/mistakes. The offenders are:

```
 grep -r FIX * | grep get_param | grep -i bug
config_src/drivers/mct_cap/mom_surface_forcing_mct.F90:  call get_param(param_file, mdl, "FIX_USTAR_GUSTLESS_BUG", CS%fix_ustar_gustless_bug, &
config_src/drivers/nuopc_cap/mom_surface_forcing_nuopc.F90:  call get_param(param_file, mdl, "FIX_USTAR_GUSTLESS_BUG", CS%fix_ustar_gustless_bug, &
config_src/drivers/FMS_cap/MOM_surface_forcing_gfdl.F90:  call get_param(param_file, mdl, "FIX_USTAR_GUSTLESS_BUG", CS%fix_ustar_gustless_bug, &
config_src/drivers/solo_driver/MOM_surface_forcing.F90:  call get_param(param_file, mdl, "FIX_USTAR_GUSTLESS_BUG", CS%fix_ustar_gustless_bug, &
src/core/MOM_dynamics_unsplit.F90:  call get_param(param_file, mdl, "FIX_UNSPLIT_DT_VISC_BUG", CS%use_correct_dt_visc, &
src/core/MOM_dynamics_unsplit_RK2.F90:  call get_param(param_file, mdl, "FIX_UNSPLIT_DT_VISC_BUG", CS%use_correct_dt_visc, &
```

**Comment by ezaron** [comment]
Can you please clarify the desired behavior and default?

Do you want `<NAME_OF_BUG>_BUG = True`  to maintain the original pre-bugfix behavior?

Do you suggest default value, `False`, so that the default fixes the bug? Or should this be decided on a case-by-case basis?

Do you want to omit the prefix "FIX_", in favor of keeping "_BUG" to make grepping easier and more consistent?

**Comment by Hallberg-NOAA** [comment]
This issue has been addressed with dev/gfdl MOM6 PRs #535 and #536, and can now be closed.
